nautobot/dcim/templates/dcim/cable_connect.html

```diff
@@ -166,3 +166,17 @@ <h3>{% block title %}Connect {{ termination_a.device }} {{ termination_a }} to {
     {% endwith %}
 </form>
 {% endblock %}
+
+{% block javascript %}
+{{ block.super }}
+<script>
+$(() => {
+    $('#select2-id_termination_b_device-container').on("click", () => {
```

Comment:
I think you'd get better behavior with something like `$('select#id_termination_b_device').on("change", ...` so that it's not clicking on the select2 element, but actually changing the selected device, that clears the termination-b dropdown.

Also, should we do the same when changing any of the other termination-b dropdowns as well (site, region, powerpanel, circuit, etc.)?

Reply:
Will add the other fields to the event listener and test using the `change` event instead 👍